third_party/move/tools/move-smith/README.md

```diff
@@ -0,0 +1,136 @@
+# MoveSmith
+
+A Move source code level fuzzer.
+
+# Usage
+
+Make sure you have all the necessary [dependencies](#dependencies).
+
+## Fuzzing
+
+To start fuzzing, run
+```bash
+cargo fuzz run <fuzz_target>
+```
+
+All fuzz targets live in `fuzz/fuzz_targets`.
```

Comment:
Could you add a bit more context on fuzz targets, and maybe a small sentence on each available option?

Reply:
Done